### ISSUE TITLE
Where can one get the latest firmware, for use on Linux?

### ISSUE THREAD

**Issue by DavidGriffith** [DavidGriffith/minipro-import-test#212]
In GitLab by @durgadas311 on Feb 20, 2021, 13:42

I don't have a Windows machine to run the XGecu app, and wanted to use the -F option with minipro. But I can't find a copy of the firmware file to download. Where can I get the necessary firmware file to use?

**Comment by DavidGriffith** [comment]
In GitLab by @radiomanV on Feb 20, 2021, 14:58

Please read here: issue #185.   
Use this link in wget command to obtain the latest firmware version: `https://github.com/Kreeblah/XGecu_Software/raw/master/Xgpro/10/xgproV1075_setup.rar`   
This will warn you about the firmware is newer than expected but just ignore it.

**Comment by DavidGriffith** [comment]
In GitLab by @durgadas311 on Feb 20, 2021, 15:54

Thanks, I had missed that issue when I searched - perhaps I neglected to un-check "OPEN issues". I had previously downloaded that same package, but "unrar l" listed only the EXE file so I thought it was not what I needed. I guess the update.dat file is hidden.

FYI, I used "unrar" and got reasonable results:
```
unrar e ./* && unrar e ./*.exe -D updateII*
```
